src/compiler/symb-define.bas

```diff
@@ -85,7 +85,11 @@ end function
 private function hDefOptGosub_cb ( ) as string
 	function = str( env.opt.gosub = TRUE )
 end function
-
+															
+private function hDefViewGui_cb ( ) as string            '***** console/gui
+	function = str( env.clopt.modeview = FB_MODEVIEW_ISGUI )
```

Comment:
Is a new option needed?  This call back could check the value of fbc.subsystem for "gui" indicator.

Reply:
please clear that request, i'm sendind a cleaned right now
thanks for supporting my wrong attempt